packages/tools/src/utilities/voi/windowlevel/extractWindowLevelRegionToolData.ts

```diff
@@ -1,11 +1,13 @@
+import type { Types } from '@cornerstonejs/core';
 import {
-  VolumeViewport,
   utilities as csUtils,
-  cache,
   StackViewport,
+  VolumeViewport,
 } from '@cornerstonejs/core';
 
-function extractWindowLevelRegionToolData(viewport) {
+function extractWindowLevelRegionToolData(
+  viewport: VolumeViewport | StackViewport
```

Comment:
This should be a type, IVolumeViewport | IStackViewport i guess

Reply:
Yes it can be.  I thought the stricter type of `VolumeViewport | StackViewport` was more appropriate because the function itself throws an error if `viewport` is not an instance of `VolumeViewport` or `StackViewport`.  If the argument type accepted `IVolumeViewport | IStackViewport` it would be theoretically possible to pass something else, causing an error to be thrown. Hence the stricter type.

Reply:
Hi, just checking if you have a preference between the stricter `viewport: VolumeViewport | StackViewport` type or `viewport: IVolumeViewport | IStackViewport`.  I can update the PR to suit

Reply:
I remember a lot of issues with using classes as types with API Extractor in the past. It was not smart enough to use types for them, and I think the rest of the app uses types, which, in fact, are the same. If you look inside IVolumeViewport, it says:

```
import type { VolumeViewport } from '../RenderingEngine';

type IVolumeViewport = VolumeViewport;

export type { IVolumeViewport as default };
```

So I think I prefer IVolumeViewport.

Reply:
No problems.  Updated.
James